labels.json

```diff
@@ -1996,6 +2021,11 @@
                 "lang": "en",
                 "name": "Mightyena",
                 "description": "This user is a Mightyena! [Category: Pokemon]"
+            },
+            {
+                "lang": "pt-BR",
+                "name": "Mightyena",
+                "description": "Este usuário é um Mightyena! [Categoria: Pokemon]  AKA: Mightyena"
```

Comment:
Please keep as `Category` as this is parsed out on https://sonasky-browse.bunnys.ky/ (I need to update the README to include this request, that's my bad)